lld/test/ELF/linkerscript/nocrossrefs.test

```diff
@@ -50,6 +54,17 @@ NOCROSSREFS(.text .text1 .text2 .data .rodata .data .nonalloc)
 abs = 42;
 NOCROSSREFS_TO(.text .text .text1 .text2 .data .nonalloc)
 
+#--- overlay.t
```

Comment:
Could be worth adding : NOCROSSREFS AT(<addr>) as a regression test.

Reply:
I think you meant `AT() NOCROSSREFS`? Done, but in `overlay.test`, since it's more of an overlay parse error than a test of `NOCROSSREFS` functionality.